server/serverAPI/controller/BaseUserController.ts

```diff
@@ -0,0 +1,62 @@
+import { Request, Response } from "express";
```

Comment:
Now that you've gotten rid of the other User types, is this base controller needed?